### ISSUE TITLE
AWSbilling failed for 3 days and still showed "STEADY" state the whole time

### ISSUE THREAD

**Issue by DmitryLitvintsev** [HEPCloud/decisionengine_modules#198]
Due to an unexpected yum update the "s3transfer" function was unavailable on hepcsvc03.
No files got downloaded from Mar 10 to Mar 13 on that system.  Yet the AWSbilling channel
showed as STEADY that whole time, even after several restarts.

So here is a suggestion of an issue that really *should* take the channel offline 
If there are actually code modules freaking missing and the code can't execute and is pushing 
out a blank data block (and not data blocks with no rows), then channel should go offline.

**Comment by DmitryLitvintsev** [comment]
Steve,
Is there any example of error due to this, like debug log?
I do not see S3Transfer used anywhere in the code
[litvinse@orbis decisionengine_modules]$ find ./ -name '*.py' -exec egrep -iH 's3transfer' {} ;
[litvinse@orbis decisionengine_modules]$

**Comment by DmitryLitvintsev** [comment]
imported from GitLab: https://hepcloud-git.fnal.gov:8443/hepcloud/decisionengine_modules/issues/170

**Comment by StevenCTimm** [comment]
s3transfer is a second or third order dependency of the BillingInfo.py code.
The dependency sequence goes BillingInfo.py -> boto3 -> s3transfer
Can replicate this by running only the AWSBilling channel
and then pip uninstall s3transfer while it is running.

**Comment by hyunwoo18** [comment]
I decided to work on this issue, actually I already started working on it.
I can not find a button to re-assign this to myself.

**Comment by hyunwoo18** [comment]
I reviewed the following codes:
`
decisionengine_modules/AWS/sources/BillingInfo.py
decisionengine/taskmanager/TaskManager.py
`
to learn how the task managers and RPCserver works
and conducted lots of testings in one of Steve's testing VM.

I think I discovered which codes need to be updated to produce the following result:
de-client --status | grep state
channel: AWSbilling, id = 09...., state = OFFLINE

I tested this in the VM after doing 
pip uninstall s3transfer
and saw state = OFFLINE
for AWSbilling channel.

Tomorrow, I will create a new branch and open a new review.
